Control.FullScreen.css

```diff
@@ -1,10 +1,39 @@
-.fullscreen-icon { background-image: url(icon-fullscreen.svg); background-size:26px 52px; }
-.fullscreen-icon.leaflet-fullscreen-on { background-position:0 -26px; }
-.leaflet-touch .fullscreen-icon { background-position: 2px 2px; }
-.leaflet-touch .fullscreen-icon.leaflet-fullscreen-on { background-position: 2px -24px; }
-/* one selector per rule as explained here : http://www.sitepoint.com/html5-full-screen-api/ */
-.leaflet-container:-webkit-full-screen { width: 100% !important; height: 100% !important; z-index: 99999; }
-.leaflet-container:-ms-fullscreen { width: 100% !important; height: 100% !important; z-index: 99999; }
-.leaflet-container:full-screen { width: 100% !important; height: 100% !important; z-index: 99999; }
-.leaflet-container:fullscreen { width: 100% !important; height: 100% !important; z-index: 99999; }
-.leaflet-pseudo-fullscreen { position: fixed !important; width: 100% !important; height: 100% !important; top: 0px !important; left: 0px !important; z-index: 99999; }
\ No newline at end of file
+.fullscreen-icon {
+  background-image: url("icon-fullscreen.svg");
+  background-size: 26px 52px;
+}
+
+.fullscreen-icon.leaflet-fullscreen-on {
+  background-position: 0 -26px;
+}
+
+.leaflet-touch .fullscreen-icon {
+  background-position: 2px 2px;
+}
+
+.leaflet-touch .fullscreen-icon.leaflet-fullscreen-on {
+  background-position: 2px -24px;
+}
+
+/* Safari still needs this vendor-prefix: https://caniuse.com/mdn-css_selectors_fullscreen */
+/* stylelint-disable-next-line selector-no-vendor-prefix */
+.leaflet-container:-webkit-full-screen {
```

Comment:
Of course You are right; let's keep the definition.

I just took the information from the [fullscreen-api](https://caniuse.com/?search=fullscreen) part of caniuse that says
>Partial support refers to not supporting ::backdrop, and supporting the old `:full-screen` syntax rather than the standard `:fullscreen`.

for "Safari 6-15.3" (i.e. for Safari till Dec 13, 2021), My interpretation was that newer versions support the `:fullscreen` pseudo selector.

I tested a version without the `:-webkit-full-screen` part on an iPad with iPadOs version 16 and it ran as expected. Therefore I decided to support only current browsers 😄 

No matter how things really are, keeping this definition will do no harm :-)